clang/tools/driver/CacheLauncherMode.cpp

```diff
@@ -105,6 +105,15 @@ static void addCommonArgs(bool ForDriver, SmallVectorImpl<const char *> &Args,
       }
     }
   };
+  auto RemotePath =
+      llvm::sys::Process::GetEnv("LLVM_CACHE_REMOTE_SERVICE_SOCKET_PATH");
+
+  if (!llvm::sys::Process::GetEnv("DISABLE_MCCAS") && !RemotePath) {
+    addCC1Args({"-fcas-backend"});
+    addCC1Args({"-fcas-backend-mode=verify"});
```

Comment:
Why is this passed by default? Doesn't it introduce overhead that we don't want at production?

Reply:
I just thought that we might want it enabled to catch any potential bugs in materialization

Reply:
It shouldn't be always on if it has overhead, I'd suggest to add `CLANG_CACHE_VERIFY_MCCAS` env variable to control enablement.

Reply:
sounds good!